st2common/st2common/util/schema/BUILD

```diff
@@ -1 +1,10 @@
-python_sources()
+python_sources(
+    dependencies=[
+        ":jsonschema",
+    ]
+)
+
+resources(
+    name="jsonschema",
+    sources=["*.json"],
+)
```

Comment:
This is required before I can submit the pants plugin that wires up pants to manage building the schemas in `contrib/schemas`.